### ISSUE TITLE
Results taking forever to show

### ISSUE THREAD

**Issue by rickmed** [appetizermonster/hain#175]
first of all, love this project! I've been wanting to build one myself for a few months..

Issue: it takes like 10 seconds to show results.
v 0.4.1

EDIT: after a 2nd restart (testing plugins) it worked much faster and including results that were lacking. 
The kill plugin never show results but that's a different issue.
This one can be closed.
EDIT: btw, there's a typo in "it may take(S) some time to show results" :)

**Comment by appetizermonster** [comment]
if you have too many files in your desktop, or start menu, maybe it will takes very long time.

**Comment by malipetek** [comment]
I can confirm that by my experience. 

I have added ${ProgramFiles}\ and ${ProgramFiles(x86)}\ paths to "Folders to search recursively" section in the "hain-plugin-filesearch"'s preferences. It became 4-5 secs to display results. So I have moved these paths to "Folders to search not recursively" section. And it is fast like fresh installed. 

Also I can find program folder like game folders in program files. But not likely subfolders of these.
